src/types.rs

```diff
@@ -24,5 +26,105 @@ pub struct TransactionInput {
 
 #[derive(Serialize, Deserialize)]
 pub struct Response {
-    pub method: Option<Function>,
+    pub method: Option<ResponseMethod>,
```

Comment:
why is it `Option`? I can't find where it would be `None`

Reply:
if it is fallback function we return ```{"method": null}```